app/views/account/tenants/index.html.slim

```diff
@@ -1,8 +1,14 @@
-h1 All tenants
-= link_to "Add new tenant", new_account_tenant_path, class: 'btn btn-primary'
-
-ul
-  - @tenants.each do |tenant|
-    li
-      = link_to tenant.name, account_tenant_path(tenant)
-= paginate @tenants
\ No newline at end of file
+.container
+  .row
+    .col-md-12
+      h1 All tenants
+      table.table.table-bordered
+        thead
+          th.col-md-1 ID
+          th.col-md-3 Name
+          th.col-md-2 Website
+          th.col-md-1 Number of users
+          th.col-md-3 Added
+          th.col-md-2 Owner
```

Comment:
remove owner column. Owner currently not works at all. We will come back to this later